Directory.Build.props

```diff
@@ -19,7 +19,8 @@
 
   <Import Project="Sdk.props" Sdk="Microsoft.DotNet.Arcade.Sdk" Condition="'$(MonoBuild)' == ''" />
   <!-- Still use global versions even if not using arcade. -->
-  <Import Project="eng/Versions.props" Condition="'$(MonoBuild)' != ''" />
+  <Import Project="$(RepositoryEngineeringDir)Versions.props" Condition="'$(MonoBuild)' != ''" />
+  <Import Project="$(RepositoryEngineeringDir)Analyzers.props" />
```

Comment:
The mono build is failing because RepositoryEngineeringDir isn't defined (since it doesn't build with Arcade).

Reply:
Yep, fixed